taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/outbox/rest/controller/CamundaTaskEventsController.java

```diff
@@ -94,16 +99,18 @@ public Response decreaseRemainingRetriesAndLogError(
   @PATCH
   @Consumes(MediaType.APPLICATION_JSON)
   @Produces(MediaType.APPLICATION_JSON)
-  public Response setRemainingRetries(
-      @PathParam("eventId") int eventId, Map<String, Integer> newRemainingRetries)
-      throws InvalidArgumentException, CamundaTaskEventNotFoundException {
+  public Response setRemainingRetries(@PathParam("eventId") int eventId, String newRemainingRetries)
```

Comment:
Maybe we should rename 'newRemainingRetries' to 'body'? 'newRemainingRetries' sound like an Integer value.

Reply:
Done